pandoc-docker.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+
+# Install dependencies for successful PDF generations
+tlmgr install ctex enumitem float koma-script
+
+# Generate PDFs using pandoc
+for filename in pandoc-*yaml; do
```

Comment:
Enhancement for a future PR - could consolidate the code in this file and `pandoc.sh`, to remove duplication. Then just pass an argument/flag to toggle docker/local calls.

Although this looks good as is.

Reply:
@jackson15j Thank you for the review! I'd actually started with functions for each. The challenge and the reason it was split is because the script for the Docker container uses a different shell invocation due to what is available within the container (`/bin/bash` vs `/bin/sh`). We could investigate other container options that ship with `/bin/bash` and the pandoc/tex tools and simplify things but it should work as is if someone were to want to clone the repo and run the pandoc script locally.